### PR TITLE
[script.game.netwalk] 0.0.6 - added add-on to games category

### PR DESCRIPTION
**Pull request by da-anda**
### Description
Since we have a game category in the add-on manager as of Leia, I thought I'd add this game to it as well.
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document

### CODE REVIEW

File: script.game.netwalk/addon.xml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>
-<addon id="script.game.netwalk" name="Netwalk Game" version="0.0.5" provider-name="Tristan Fischer (sphere@dersphere.de)">
+<addon id="script.game.netwalk" name="Netwalk Game" version="0.0.6" provider-name="Tristan Fischer (sphere@dersphere.de)">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```
Comment (Rechi) [inline script.game.netwalk/addon.xml]: `WARN: For gotham it is advised to set xbmc.python version to 2.14.0`

**Comment by da-anda** [comment]
got a reply from @dersphere and he's fine with the PR (and me taking over the add-on, but we'll see about that)

**Comment by dersphere** [comment]
Yes I totally fine with it 👍

**Comment by enen92** [comment]
Please remove empty tags to keep travis happy (language - which only refers to content addons, and website)

File: script.game.netwalk/addon.xml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>
-<addon id="script.game.netwalk" name="Netwalk Game" version="0.0.5" provider-name="Tristan Fischer (sphere@dersphere.de)">
+<addon id="script.game.netwalk" name="Netwalk Game" version="0.0.6" provider-name="Tristan Fischer (sphere@dersphere.de)">
     <requires>
-        <import addon="xbmc.python" version="2.1.0"/>
+        <import addon="xbmc.python" version="2.14.0"/>
     </requires>
-    <extension point="xbmc.python.script" library="game.py"/>
+    <extension point="xbmc.python.script" library="game.py">
+        <provides>game</provides>
```
Comment (Rechi) [inline script.game.netwalk/addon.xml]: @da-anda have you tested if this works on older Kodi/XBMC versions?

Comment (da-anda) [reply script.game.netwalk/addon.xml]: will double check when I work on those two extensions again. Been a bit busy getting the terrace/deck done before temperature drops below zero at night

**Comment by Rechi** [comment]
@da-anda this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Pull request merged**